mod_Kinematics.F90

```diff
@@ -0,0 +1,8210 @@
+MODULE ModKinematics
```

Comment:
This file seems to have been written from scratch, Would you please merge it somehow and distill the changes?

Reply:
I have deleted this rogue file and also another one that somehow slipped in the commit. Sorry about that.